include/matioCpp/Element.h

```diff
@@ -105,6 +105,16 @@ class matioCpp::Element : public matioCpp::Variable
      */
     Element<T>& operator=(element_type value);
 
+    /**
+     * @brief Get this Vector as a Span
+     */
+    matioCpp::Span<element_type> toSpan();
+
+    /**
+     * @brief Get this Vector as a Span (const version)
+     */
+    const matioCpp::Span<const element_type> toSpan() const;
```

Comment:
Nitpick, i.e. feel free to ignore: for methods that just provide a different "view" of the existing object/memory, I tipically find "as<>" methods (so in this case `asSpan`) more descriptive.

Reply:
I have used ``as*`` for the downcasting of a ``Variable`` object to its derived, while I used ``to*`` for other types (like [``toMatio`` ](https://github.com/ami-iit/matio-cpp/blob/1555d2451dbc56665b89d82028cd3225b24d0e82/include/matioCpp/Variable.h#L283) and ``toSpan``). To be honest, I am not sure this difference was completely intentional at the beginning, but I would keep it as is for the time being. The main reason I added ``toSpan`` to ``Element`` was to have the possibility of using that method in templated functions accepting also ``Vector`` and ``MultiDimensionalArray``, where the ``toSpan`` method is already defined.